packages/frontend/src/components/creator/CreatorPoolParamsDialog.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  Button,
+  FormControl,
+  FormLabel,
+  Input,
+  InputGroup,
+  InputRightAddon,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+  useToast,
+} from '@chakra-ui/react'
+import { useDeployments } from '@lib/useDeployments'
+import { BeneficiaryPool__factory } from '@yieldgate/contracts/typechain-types'
+import { ethers } from 'ethers'
+import { FC, useRef, useState } from 'react'
+import { useSigner } from 'wagmi'
+
+export interface CreatorPoolParamsDialogProps {
+  isOpen: boolean
+  onClose: () => void
+  poolAddress?: string | false
+  minAmount?: string
+  minDurationDays?: number
+  refetchPoolParams: () => void
+}
+export const CreatorPoolParamsDialog: FC<CreatorPoolParamsDialogProps> = ({
+  isOpen,
+  onClose,
+  poolAddress,
+  minAmount: fetchedMinAmount,
+  minDurationDays: fetchedMinDurationDays,
+  refetchPoolParams,
+}) => {
+  const initialRef = useRef(null)
+  const { data: signer, refetch: refetchSigner } = useSigner()
+  const { contracts, contractsChain } = useDeployments()
+  const [minAmount, setMinAmount] = useState<string>()
+  const [minDurationDays, setMinDurationDays] = useState<number>()
+  const [isLoading, setIsLoading] = useState(false)
+  const toast = useToast()
+
+  const updatePoolParameters = async () => {
+    await refetchSigner()
+    if (!poolAddress || !contracts || !signer) return
+    setIsLoading(true)
+
+    try {
+      const poolContract = BeneficiaryPool__factory.connect(poolAddress, signer)
+      const minDurationSeconds =
+        (minDurationDays ?? Math.floor(fetchedMinDurationDays || 0)) * 24 * 60 * 60
```

Comment:
Couldn't we allow fractional days? Like
```suggestion
        (minDurationDays ?? (fetchedMinDurationDays || 0)) * 24 * 60 * 60
```
Maybe also use `dayjs` here? They have a [duration extension](https://day.js.org/docs/en/plugin/duration).

Reply:
Yeah, we could do that. I wanted to keep it simple as fractional days don't make too much sense IMO (what staking amount can even be generated in these short time frames). 

I didn't use the `dayjs` extension just because it's another 5 KB dependency for one single rather simple multiplication. If we need this one more places or for other units in the future I would also prefer it.

Reply:
I also see fractional days as low prio, happy to skip it.

Re `dayjs`, you already use it in files `Post.tsx`, `creator/SupporterStakeDialog.tsx` and `creator/CreatorCardActions.tsx`, or would using it here still bloat the deps?

Reply:
The extension would be an extra dep, as it's not part of the core. They designed it this way to keep dayjs itself very slim.